### DISCUSSION TITLE
Mini vs Full version - What are the differences?

### DISCUSSION THREAD

**Discussion by Anutrix** [cmderdev/cmder#2965] [Q&A]
This is a duplicate of https://github.com/cmderdev/cmder/issues/1960 but since I can't post on it I created new ticket.

### Description of the issue

When visiting download page, the full version says - _with Git for Windows_ while mini mentions nothing.
So I assumed Full is just Mini+Git for Windows.

Then I see in the `Git and others` section:
> Oooh yes! If you decide to use the slightly bigger[ git-for-windows](https://gitforwindows.org/) version, you will have all Unix commands ready in PATH so that you can git init or cat instantly on every machine.

The words **slightly bigger** adds to the doubt whether Full version only has Git as extra item. After all, ~121MB is not just slightly bigger than ~10.0MB.

I would suggest adding at least a line near the two download buttons stating clearly "Full version is just Mini version+Git for Windows". And also the remove the word *slightly* from the `Git and others` section.

**Comment by daxgames** [comment]
The only diff between mini and full is Git for Windows and I think the docs are pretty clear on that fact.

**Comment by DRSDavidSoft** [comment]
@Anutrix Good suggestion, although this is related to the [website repo](https://github.com/cmderdev/cmderdev.github.io) and is a remnant of when the full version actually _was_ slightly bigger. The comparison was also made between the higher compression 7-zip file and the mini zip file.

I believe we could benefit from a more clear and concise description for the differences between the Full version and Mini version on the website.

**Comment by daxgames** [comment]
@Anutrix 'Cmder - Full' is EXACTLY what the website says.  It is 'Cmder Mini + Git for Windows'.

In addition to `git` Git for Windows happens to ALSO include a relatively full set of Unix like tools found on BSD, Linux and Unix, and it alone adds ~121 MB to the 'Cmder -Full' archive.

**Comment by maphew** [comment]
For me at this moment on the website (https://cmder.app/) I can't find the phrase 'Cmder Mini + Git for Windows'. I see _"Download Full"_ and _"(with Git for Windows)"_. To me 'with' implies there may be other differences as well, it's one of a possible set. 

My reading and search path trying to answer this question before tagging on to this issue was: _Website >> Repo Readme >> Wiki >> Discussion ([ref](https://github.com/orgs/cmderdev/discussions/2865))  >> here._

**Reply by maphew** [reply]
a question behind the question is: if a person already has Git for Windows installed will there be problems using the Cmder Full release? and: How does one integrate Cmder Mini with an existing Git install? [Integration](https://github.com/cmderdev/cmder/wiki#cmder-integration) in the wiki doesn't cover this scenario.

**Reply by maphew** [reply]
Ahh, there it is: https://github.com/cmderdev/cmder/blob/master/README.md#vendored-git

**Comment by jackusay** [comment]
I just find out cmder full version including `git gui`, so I don't need to download `git for windows` for gui...
You can launch it by typing `git gui` in CLI.
